src/framework/mlt_consumer.c

```diff
@@ -852,8 +852,10 @@ static void *consumer_read_ahead_thread( void *arg )
 		pthread_cond_broadcast( &priv->queue_cond );
 		pthread_mutex_unlock( &priv->queue_mutex );
 
+		mlt_log_timings_begin();
 		// Get the next frame
 		frame = mlt_consumer_get_frame( self );
+		mlt_log_timings_end(NULL, "mlt_consumer_get_frame");
```

Comment:
Should the service be provided instead of NULL?

The style in this file is to have space after (

`mlt_log_timings_end( self, "mlt_consumer_get_frame" );`

Same comment for the next two instances.